### PR TITLE
Ukrainian translation

### PR DESCRIPTION
**Pull request by Cladis**
Ukrainian translation (from English) + typo fix in English one

### CODE REVIEW

**Comment by manicki** [comment]
Wow, thanks! Could you please split the typo fix from this to a separate pull request? It will just make review etc easier

**Comment by manicki** [comment]
Oh, and just in cases you did not notice that: complete translation consists of the messages in the JSON file you've translated, and there are also four HTML files in each subdirectory of `i18n`: `about.html`, `feedback.html`, `legal.html`, `private-use.html`

File: i18n/uk/i18n.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  "info-box": {
+    "private-use": "Для більшості приватних використань Вам не потрібно вказувати ліцензію.",
+    "private-use-more-link": "&#8594; ДІЗНАТИСЯ БІЛЬШЕ",
+    "dont-show-again": "Не показувати це повідомлення знову.",
+    "understood-and-close": "Зрозуміло, закрити.",
+    "ported-licence": "Деякі ліцензії Creative Commons було опубліковано в редакціях для певних країн, так званих «адаптованих ліцензіях». Зображення, для якого Ви запитували вказання ліцензії, опубліковане з такою адаптованою ліцензією. Інструмент не може перевірити, чи ліцензійні вимоги цієї ліцензії Creative Commons для окремої країни еквівалентні англомовній версії. Однак Ви все одно можете згенерувати повідомлення-атрибуцію про ліцензію цим інструментом. Щоб бути повністю певними, ми радимо перевірити вимоги цієї версії ліцензії для країни. Додатково це вказання ліцензії повинне бути мовою її вмісту, щоб виконувати свою функцію повністю."
+  },
+  "dialogue": {
+    "public-domain-picture": "При завантаженні це зображення було позначене як таке, що перебуває у суспільному надбанні, тож Вам не потрібно вказувати ліцензію.",
+    "no-attribution-needed": "Вказання ліцензії не потрібне,
```
Comment (manicki) [inline i18n/uk/i18n.json]: Missing closing quote here. This makes the whole JSON file not well-formed, and app cannot pick it up.

**Comment by Cladis** [comment]
Ok, I took a while to do anything. Well, I have no clue how to split it into several PRs :( I have fixed the missed quotation though, thank you for finding the needle.
Yeah, I am aware about the HTML files, I planned to translate them later.

**Comment by manicki** [comment]
Thanks a lot @Cladis, it was a while!
Splitting off the typo fix was no longer needed, so this was all good.
Feel free to add translation of HTML files whenever you're ready!

**Comment by manicki** [comment]
Hi @Cladis, I have tested the Ukrainian version before putting it online (I think it would be still fine even with those HTML files not yet translated).
I've noticed that some UI texts are not translated. This is due to our fault. There were some minor changes done to Attribution Generator recently, and there are couple of new i18n messages to translate.

While I am able to somewhat read Ukrainian, I would rather not try to add missing translations myself. Could you maybe translate these below, and I would add them to the json file and publish the UK translation online?

Message key | German text | English text | Notes
-----------------|-------------|-------------|---------
feedback-response-email | E-Mail Address | E-Mail-Adresse | Field in the feedback form
back-to-top | Weiteres Bild prüfen | Check another image | Button shown after generating attibution
force-pd-attribution | Trotzdem Rechtehinweis erstellen | Create a legal notice anyway | Button visible when input is a PD-licensed image
change-pd-headline | Wenn Sie möchten, können Sie Ihre Änderungen kenntlich machen. Was genau wird am Bild verändert? | Which part of the image will be modified? | Visible when generating attribution for PD image, and the image was modified
mark-as-pd | als gemeinfrei markieren | mark as public domain | License option visivle when generating attribution for the PD image, in the license step

Sorry for extra work and thanks!

**Pull request merged**